I have the trouble (on Cheyenne) in the ecbuild step for CRTM with the error message:
```
-- Adding bundle project crtm

-- Cloning crtm from https://github.com/JCSDA/crtm.git into /glade/scratch/liuz/public_jcsda/code/crtm...

-- /glade/scratch/liuz/public_jcsda/code/crtm retrieved.

-- Updating crtm to head of BRANCH release/crtm_jedi...

-- git fetch --all @ /glade/scratch/liuz/public_jcsda/code/crtm

-- git fetch --all --tags @ /glade/scratch/liuz/public_jcsda/code/crtm

-- git checkout release/crtm_jedi @ /glade/scratch/liuz/public_jcsda/code/crtm

-- ---------------------------------------------------------

-- [crtm] (2.3.0) [bfede42]

-- Feature TESTS enabled

CMake Error at /glade/scratch/liuz/public_jcsda/build/jedicmake/jedicmake-config.cmake:19 (message):

  File or directory

  /glade/scratch/liuz/public_jcsda/code/jedicmake/CMakeModules/Modules

  referenced by variable jedicmake_MODULES does not exist !

Call Stack (most recent call first):

  /glade/scratch/liuz/public_jcsda/build/jedicmake/jedicmake-config.cmake:36 (set_and_check)

  crtm/CMakeLists.txt:20 (find_package)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,7 +21,7 @@ find_package( ecbuild 3.5 REQUIRED HINTS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CUR
 include( ecbuild_bundle )
 ecbuild_bundle_initialize()
 
-ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/JCSDA/jedi-cmake.git" BRANCH develop UPDATE )
+ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/JCSDA/jedi-cmake.git" BRANCH develop UPDATE RECURSIVE)
 include( jedicmake/cmake/Functions/git_functions.cmake )
 
 option(BUNDLE_SKIP_ECKIT "Don't build eckit" "ON" ) # Skip eckit build unless user passes -DBUNDLE_SKIP_ECKIT=OFF

